platform/android/java/lib/src/org/godotengine/godot/xr/regular/RegularContextFactory.java

```diff
@@ -51,8 +51,6 @@
 	private static int EGL_CONTEXT_CLIENT_VERSION = 0x3098;
 
 	public EGLContext createContext(EGL10 egl, EGLDisplay display, EGLConfig eglConfig) {
-		String driver_name = GodotLib.getGlobal("rendering/quality/driver/driver_name");
-		// FIXME: Add support for Vulkan.
```

Comment:
Is this really solved though?

Reply:
I figured handling Vulkan wasn't going to take place in this file since it's GL specific, but I can put the comment back if you want.